dcs/statics.py

```diff
@@ -462,6 +462,618 @@ class Beer_Bomb(unittype.StaticType):
+        rate = 1
+
+    class Cargo01(unittype.StaticType):
+        id = "Cargo01"
+        name = "M92 Cargo 01"
+        shape_name = "M92_Cargo01"
+        rate = 1
+
+    class Cargo02(unittype.StaticType):
+        id = "Cargo02"
+        name = "M92 Cargo 02"
+        shape_name = "M92_Cargo02"
+        rate = 1
+
+    class Cargo03(unittype.StaticType):
+        id = "Cargo03"
+        name = "M92 Cargo 03"
+        shape_name = "M92_Cargo03"
+        rate = 1
+
+    class Cargo04(unittype.StaticType):
+        id = "Cargo04"
+        name = "M92 Cargo 04"
+        shape_name = "M92_Cargo04"
+        rate = 1
+
+    class Cargo05(unittype.StaticType):
+        id = "Cargo05"
+        name = "M92 Cargo 05"
+        shape_name = "M92_Cargo05"
+        rate = 1
+
+    class Cargo06(unittype.StaticType):
+        id = "Cargo06"
+        name = "M92 Cargo 06"
+        shape_name = "M92_Cargo06"
+        rate = 1
+
+    class Camouflage01(unittype.StaticType):
+        id = "Camouflage01"
+        name = "M92 Camouflage 01"
+        shape_name = "M92_Camouflage01"
+        rate = 1
+
+    class Camouflage02(unittype.StaticType):
+        id = "Camouflage02"
+        name = "M92 Camouflage 02"
+        shape_name = "M92_Camouflage02"
+        rate = 1
+
+    class Camouflage03(unittype.StaticType):
+        id = "Camouflage03"
+        name = "M92 Camouflage 03"
+        shape_name = "M92_Camouflage03"
+        rate = 1
+
+    class Camouflage04(unittype.StaticType):
+        id = "Camouflage04"
+        name = "M92 Camouflage 04"
+        shape_name = "M92_Camouflage04"
+        rate = 1
+
+    class Camouflage05(unittype.StaticType):
+        id = "Camouflage05"
+        name = "M92 Camouflage 05"
+        shape_name = "M92_Camouflage05"
+        rate = 1
+
+    class Camouflage06(unittype.StaticType):
+        id = "Camouflage06"
+        name = "M92 Camouflage 06"
+        shape_name = "M92_Camouflage06"
+        rate = 1
+
+    class Camouflage07(unittype.StaticType):
+        id = "Camouflage07"
+        name = "M92 Camouflage 07"
+        shape_name = "M92_Camouflage07"
+        rate = 1
+
+    class Cone01(unittype.StaticType):
+        id = "Cone01"
+        name = "M92 Cone 01"
+        shape_name = "M92_Cone01"
+        rate = 1
+
+    class Cone02(unittype.StaticType):
+        id = "Cone02"
+        name = "M92 Cone 02"
+        shape_name = "M92_Cone02"
+        rate = 1
+
+    class Container_10ft(unittype.StaticType):
+        id = "Container_10ft"
+        name = "M92 Container 10ft"
+        shape_name = "M92_Container_10ft"
+        rate = 1
+
+    class Container_20ft(unittype.StaticType):
+        id = "Container_20ft"
+        name = "M92 Container 20ft"
+        shape_name = "M92_Container_20ft"
+        rate = 1
+
+    class Container_40ft(unittype.StaticType):
+        id = "Container_40ft"
+        name = "M92 Container 40ft"
+        shape_name = "M92_Container_40ft"
+        rate = 1
```

Comment:
These two are conflicting with existing "containers". Notice how the "id" is different is casing compared to the ones starting at line 1167.
At any rate, one way to fix this is is to avoid capitalizing the first letter of the class, but that'll change quite a few other vehicles as well which _can_ cause a headache when updating pydcs in Liberation 😬

Reply:
Yeah, I saw your commits in the Retribution fork of pydcs. I went a different way of suffixing duplicates with an underscore, which means there isn't any change to existing unit classes. Probably doesn't make a huge difference for Liberation / Retribution but at the end of the day pydcs is a library and keeping class names unchanged (as much as DCS data allows) would be good.

Reply:
Agreed, that's definitely the better option in hindsight 😅